proto/nexus/exported/v1beta1/types.proto

```diff
@@ -52,11 +54,22 @@ message TransferFee {
   ];
 }
 
-message Asset {
-  string denom = 1;
-  bytes min_amount = 2 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+message FeeInfo {
+  bytes fee_rate = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  bytes min_fee = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Uint",
+    (gogoproto.nullable) = false
+  ];
+  bytes max_fee = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Uint",
```

Comment:
I don't like adding redundant denom info to each field as we need to be extra careful that the values are consistent, along with the fact that `Coin` uses `Int`. But I can be convinced of otherwise.

Reply:
but if you pass the fee info around you don't know the denomination anymore. I can see your argument, so I'd like @jcs47, @sammy1991106 and @haiyizxx to chime in here as well

Reply:
why not keep the current fields and had an extra one to explicitly describe the denom?

Reply:
Hmmm if we are talking about passing this feeInfo around, then both asset and chain need to be in there.

Reply:
Will do that